Screen-Sharing-Video-Chat /js/config.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+// Make a copy of this file and save it as config.js (in the js directory).
+
+// Set this to the base URL of your sample server, such as 'https://your-app-name.herokuapp.com'.
+// Do not include the trailing slash. See the README for more information:
+
+var SAMPLE_SERVER_BASE_URL = 'http://YOUR-SERVER-URL';
```

Comment:
Please use `const` instead of `var`. We should be using es6, we need to update the whole repo, but welcome to do it if you are willing :)

Reply:
fixed 👍

Reply:
This change has been reverted as per @jeffswartz 's request. This was made to allow the sample to work in IE11